ansible_runner/interface.py

```diff
@@ -366,14 +366,14 @@ def run_command(executable_cmd, cmdline_args=None, **kwargs):
     :type status_handler: function
     :type artifacts_handler: function
 
-    :returns: Retunes a tuple of response and error string. In case if ``runner_mode`` is set to ``pexpect`` the error value is empty as
+    :returns: Retunes a tuple of return code, response and error string. In case if ``runner_mode`` is set to ``pexpect`` the error value is empty as
```

Comment:
```suggestion
    :returns: Returns a tuple of return code, response and error string. In case if ``runner_mode`` is set to ``pexpect`` the error value is empty as
```